src/content/structured/components/checkboxes/code.mdx

```diff
@@ -30,21 +30,40 @@ export const snippetsDefault = [
   <ic-checkbox value="extra" label="Extra shot (50p)"></ic-checkbox>
   <ic-checkbox value="Soya milk" label="Soya milk" checked ></ic-checkbox>
   <ic-checkbox value="keep cup" label="Takeaway cup (Reusable keep cups only)" disabled="true" ></ic-checkbox>
-</ic-checkbox-group>`,
+</ic-checkbox-group>
+<script>
+  const checkboxGroup = document.querySelector('ic-checkbox-group');
+  checkboxGroup.addEventListener('icChange', (ev) => console.log(ev.detail.value));
```

Comment:
```suggestion
  checkboxGroup.addEventListener('icChange', (ev) => console.log("onIcChange", ev.detail.value));
```
to match preview example

Reply:
Updated this one to `icChange` as it's the web-components example